### ISSUE TITLE
add seq2seq layer 同

### ISSUE THREAD

**Issue by shelfee** [farizrahman4u/seq2seq#270]
> `    model = Sequential()
    model.add(Embedding(39, 39, input_length=24, mask_zero=True))
    model.add(Seq2Seq(input_length=24, input_dim=39, hidden_dim=10, output_length=22, output_dim=39))`
raise the Exception

**Comment by ketyi** [comment]
I've the exact same Exception with an ultra simple:
`base_model = Seq2Seq(input_dim=128, input_length=150, hidden_dim=512, output_length=150, output_dim=128, depth=2)
    multi_model = multi_gpu_model(base_model, gpus=2)
    multi_model.compile(loss='mse', optimizer='adam', metrics=['mse'])`

I don't have any masks and with the AttentionSeq2Seq class just worked...

**Comment by foreshadow** [comment]
Same issue.

**Comment by vffuunnyy** [comment]
same.

**Comment by byamao1** [comment]
I encounter this problem too.

**Comment by Lance-Tin** [comment]
I encounter this problem too. just executed the example code;
tensorflow==1.13.1
keras==2.2.4
python==3.6.2
